samples/nrf9160/modem_shell/sample.yaml

```diff
@@ -40,14 +40,42 @@ tests:
     integration_platforms:
       - nrf9160dk_nrf9160_ns
     platform_allow: nrf9160dk_nrf9160_ns
-    extra_args: SHIELD=nrf7002_ek OVERLAY_CONFIG=overlay-nrf7002ek-wifi-scan-only.conf DTC_OVERLAY_FILE="nrf9160dk_with_nrf7002ek.overlay"
+    extra_args: SHIELD=nrf7002ek_nrf7002 OVERLAY_CONFIG=overlay-nrf7002ek-wifi-scan-only.conf DTC_OVERLAY_FILE="nrf9160dk_with_nrf7002ek.overlay"
+    tags: ci_build
+  sample.nrf9160.modem_shell.nrf7000ek_wifi:
+    build_only: true
+    integration_platforms:
+      - nrf9160dk_nrf9160_ns
+    platform_allow: nrf9160dk_nrf9160_ns
+    extra_args: SHIELD=nrf7002ek_nrf7000 OVERLAY_CONFIG=overlay-nrf7002ek-wifi-scan-only.conf DTC_OVERLAY_FILE="nrf9160dk_with_nrf7002ek.overlay"
+    tags: ci_build
+  sample.nrf9160.modem_shell.nrf7001ek_wifi:
+    build_only: true
+    integration_platforms:
+      - nrf9160dk_nrf9160_ns
+    platform_allow: nrf9160dk_nrf9160_ns
+    extra_args: SHIELD=nrf7002ek_nrf7001 OVERLAY_CONFIG=overlay-nrf7002ek-wifi-scan-only.conf DTC_OVERLAY_FILE="nrf9160dk_with_nrf7002ek.overlay"
     tags: ci_build
   sample.nrf9160.modem_shell.nrf7002ek_wifi-debug:
     build_only: true
     integration_platforms:
       - nrf9160dk_nrf9160_ns
     platform_allow: nrf9160dk_nrf9160_ns
-    extra_args: SHIELD=nrf7002_ek OVERLAY_CONFIG="overlay-nrf7002ek-wifi-scan-only.conf;overlay-debug.conf" DTC_OVERLAY_FILE="nrf9160dk_with_nrf7002ek.overlay"
+    extra_args: SHIELD=nrf7002ek_nrf7002 OVERLAY_CONFIG="overlay-nrf7002ek-wifi-scan-only.conf;overlay-debug.conf" DTC_OVERLAY_FILE="nrf9160dk_with_nrf7002ek.overlay"
+    tags: ci_build
+  sample.nrf9160.modem_shell.nrf7000ek_wifi-debug:
+    build_only: true
+    integration_platforms:
+      - nrf9160dk_nrf9160_ns
+    platform_allow: nrf9160dk_nrf9160_ns
+    extra_args: SHIELD=nrf7002ek_nrf7000 OVERLAY_CONFIG="overlay-nrf7002ek-wifi-scan-only.conf;overlay-debug.conf" DTC_OVERLAY_FILE="nrf9160dk_with_nrf7002ek.overlay"
+    tags: ci_build
+  sample.nrf9160.modem_shell.nrf7001ek_wifi-debug:
+    build_only: true
+    integration_platforms:
+      - nrf9160dk_nrf9160_ns
+    platform_allow: nrf9160dk_nrf9160_ns
+    extra_args: SHIELD=nrf7002ek_nrf7001 OVERLAY_CONFIG="overlay-nrf7002ek-wifi-scan-only.conf;overlay-debug.conf" DTC_OVERLAY_FILE="nrf9160dk_with_nrf7002ek.overlay"
```

Comment:
I agree, that's something to consider because there's a lot of nrf700x configurations now.

Maybe we should also change the naming of the related `modem_shell` overlays? But that can be done outside this PR.

Reply:
I would keep only nrf7002 version for debug and pgps configurations. But my first comment still applies that I don't know if there is a reason to compile everything with all nrf700x variants.

So I would remove these builds from modem_shell sample.yaml:
sample.nrf9160.modem_shell.nrf7000ek_wifi-debug
sample.nrf9160.modem_shell.nrf7001ek_wifi-debug
sample.nrf9160.modem_shell.location_service_ext_pgps_nrf7000ek_wifi
sample.nrf9160.modem_shell.location_service_ext_pgps_nrf7001ek_wifi